src/components/HomeViewComponents/HeaderItems.js

```diff
@@ -0,0 +1,80 @@
+import { Paper, makeStyles, Container, Box, Typography, Button, Hidden } from "@material-ui/core";
```

Comment:
Change this name from `HeaderItems` to `Hero`.
These are called Heros

Reply:
Pushing the change in this PR #345 !!